pkg/flatten/testdata/general/fail-nicely-when-no-http-client-provided_id.yaml

```
@@ -0,0 +1,11 @@
+name: "Fails nicely when no HTTP client provided and id is used"
+
+input:
+  workspace:
+    components:
+      - name: test-plugin
+        plugin:
+          id: test/plugin
+
+output:
+  errRegexp: "cannot resolve resources by id: no HTTP client provided"
```

```suggestion
  errRegexp: "cannot resolve resources by id: no HTTP client provided"

```
the value of my review increases after I commented all end-file empty lines )
Here and a few places below.